lib/nahmii-provider.spec.js

```diff
@@ -273,6 +311,8 @@ describe('Nahmii Provider', () => {
         it('can watch a tx hash and reject when it\'s mined but failed to execute', (done) => {
             const txReceipt = {hash: 'magic tx hash 2', status: 0};
             const txHash = '0x1bb332f5b3c2c6b56e43284145c1cb7454606d89bd9b82bd8821e3edfe8d35ad';
+
+            // TODO: Fix this. Dont stub the code under test!
```

Comment:
Was the intention to fix this in the current PR? Or a later one?

Reply:
No. Its just a reminder. The issue wasn't added in this PR so I will fix it later.